gnd/src/main/java/com/google/android/gnd/ui/home/mapcontainer/MapContainerViewModel.java

```diff
@@ -354,6 +432,26 @@ public void onCameraMove(CameraPosition newCameraPosition) {
             project -> projectRepository.setCameraPosition(project.getId(), newCameraPosition));
   }
 
+  public void onCompletePolygonButtonClick() {
+    updatePolygonDrawing(PolygonDrawing.STARTED);
+    setViewMode(Mode.DEFAULT);
+    featureRepository.newPolygonFeature(
+        selectedProject.getValue().get(),
+        selectedLayer.getValue().get(),
+        ImmutableList.<Point>builder().addAll(vertices).build());
```

Comment:
This can be removed. New feature is not assigned to anything

Reply:
removed will add in next PR.

Reply:
Not removed